src/axom/lumberjack/MPIUtility.cpp

```diff
@@ -49,7 +51,8 @@ void mpiNonBlockingSendMessages(MPI_Comm comm, int destinationRank,
   MPI_Request mpiRequest;
   MPI_Isend(const_cast<char*>(packedMessagesToBeSent),
             strlen(packedMessagesToBeSent), MPI_CHAR,
-            destinationRank, 0, comm, &mpiRequest);
+            destinationRank, LJ_TAG, comm, &mpiRequest);
+  MPI_Request_free(&mpiRequest);
```

Comment:
Good catch!

Reply:
@kennyweiss finally figured it out in a long won battle to get me out of his office...

Reply:
awesome you tracked down the issue!